store.go

```diff
@@ -1455,6 +1462,47 @@ func (s *store) ImageBigDataSize(id, key string) (int64, error) {
 	return -1, ErrSizeUnknown
 }
 
+func (s *store) MountTemp(id, source string) (string, error) {
+	rcstore, err := s.ContainerStore()
+	if err != nil {
+		return "", err
+	}
+	rcstore.RLock()
+	defer rcstore.Unlock()
+	container, err := rcstore.Get(id)
+	if err != nil {
+		return "", err
+	}
+	if modified, err := rcstore.Modified(); modified || err != nil {
+		if err = rcstore.Load(); err != nil {
+			return "", err
+		}
+	}
+	driver, err := s.GraphDriver()
+	if err != nil {
+		return "", err
+	}
+	middleDir := s.graphDriverName + "-containers"
+	sourcedir := filepath.Join(s.GraphRoot(), middleDir, id, strings.Replace(source, "/", "_", -1))
+
+	return driver.MountTemp(sourcedir, source, container.MountLabel())
+}
+
+func (s *store) RemoveTemp(source string) error {
+	rcstore, err := s.ContainerStore()
```

Comment:
This doesn't seem to be used in this function.

Reply:
I thought I should lock the container store while it is deleted, but if you think it is safe, I can remove this.